traffic_ops/app/lib/API/DeliveryServiceMatches.pm

```diff
@@ -28,11 +29,19 @@ sub index {
 	my $format = $self->param("format") || "";
 
 	my $rs;
-	if ( &is_privileged($self) ) {
+	# TO the reviewer: Do we need to override the "is_priviledged" here byt the standard "ignore_ds_user_table" flag?
+	# What is the reason of the is_priv test - was someone just dussmissed the ds_tmuser table tests
+	if ( &is_privileged($self)) {
```

Comment:
what about if you change this line to:

if ( (use-tenancy=1) || &is_privileged($self)) {

^^ that's just pseudo code obviously

if tenancy is in place, fine let them in and you will only see "matches" for the ds's assigned to you
if tenancy is not in place, then it will fall back to what it was before and will check is_privileged